### ISSUE TITLE
[Bug] Triggers that are moved by the cluster as the node is dead do not appear on failed triggers table

### ISSUE THREAD

**Issue by u-n-i-c-o-rn** [z1pti3/jimi#221]
Triggers that are moved by the cluster as the node is dead do not appear on failed triggers table

**Comment by z1pti3** [comment]
These actions are audited but you are correct they are not included on the status table

```
jimi.audit._audit().add("cluster","set trigger",{ "triggerID" : inactiveTrigger._id, "triggerName" : inactiveTrigger.name, "systemID" : member, "clusterSet" : inactiveTrigger.clusterSet, "masterID" : self.systemID, "masterUID" : self.systemUID })
```

Need to update search query used to generate that table to include the above audit items

**Comment by b1scuit-thi3f** [comment]
This has been fixed with ad0afe5fc45737e32f4bd850458dd5df07cc1bec
![image](https://user-images.githubusercontent.com/14958920/135727061-c2ea1d9d-a801-4d67-b646-c131631622f2.png)
